runtime/parachains/src/scheduler/tests.rs

```diff
@@ -1033,55 +1026,42 @@ fn availability_predicate_works() {
 			_ => None,
 		});
 
-		// assign some availability cores.
+		// assign some availability core.
 		{
 			let entry_ttl = 10_000;
 			AvailabilityCores::<Test>::mutate(|cores| {
-				cores[0] =
-					CoreOccupied::Paras(ParasEntry::new(Assignment::new(chain_a), entry_ttl));
-				cores[1] =
-					CoreOccupied::Paras(ParasEntry::new(Assignment::new(thread_a), entry_ttl));
+				cores[0] = Some(CoreOccupied::Parachain);
```

Comment:
We need to use the new `CoreOccupied` here :
```suggestion
				cores[0] = CoreOccupied::Paras(ParasEntry::new(Assignment::new(chain_a), entry_ttl));
```